pim-tool-front/src/app/service/group.service.ts

```diff
@@ -0,0 +1,19 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Group } from '../model/group';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class GroupService {
+  private projectUrl: string;
+
+  constructor(private http: HttpClient) {
+    this.projectUrl = 'http://localhost:8080';
```

Comment:
[N] What is the difference by setting this value in constructor and setting it directly at line 10?